WheelViewLib/src/main/java/com/lukedeighton/wheelview/WheelView.java

```diff
@@ -950,6 +955,23 @@ private void setSelectedPosition(int position) {
             int adapterPos = getSelectedPosition();
             mOnItemSelectListener.onWheelItemSelected(this, getWheelItemDrawable(adapterPos), adapterPos);
         }
+
+        informIfSettled(getSelectedPosition());
+
+    }
+
+    private void informIfSettled(final int position){
+        final WheelView wheelView = this;
+        new Handler().postDelayed(new Runnable() {
```

Comment:
Looks good - Although I'd like to avoid using a delayed runnable if possible. :thinking: